### PR TITLE
Optional flag of collect skip comments

### PR DESCRIPTION
**Pull request by tronxd**
* Optional flag to disable collection of skip checks comments

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/terraform/context_parsers/base_parser.py [hunk 1]
```diff
@@ -103,12 +103,13 @@ def _compute_definition_end_line(self, start_line_num):
                     break
         return end_line_num
 
-    def run(self, tf_file, definition_blocks):
+    def run(self, tf_file, definition_blocks, collect_skip_comments):
```
Comment (schosterbarak) [inline checkov/terraform/context_parsers/base_parser.py:106]: you probably want to add a default value here or the new param

File: checkov/terraform/context_parsers/registry.py [hunk 2]
```diff
@@ -15,7 +15,7 @@ def register(self, parser):
     def reset_definitions_context(self):
         self.definitions_context = {}
 
-    def enrich_definitions_context(self, definitions):
+    def enrich_definitions_context(self, definitions, collect_skip_comments):
```
Comment (schosterbarak) [inline checkov/terraform/context_parsers/registry.py:18]: you probably want to add a default value here or the new param

File: checkov/terraform/runner.py [hunk 3]
```diff
@@ -77,11 +77,11 @@ def evaluate_string_booleans(self):
                 if not var_path.endswith('alias/0'):
                     dpath.set(self.tf_definitions[tf_file], var_path, False)
 
-    def check_tf_definition(self, report, root_folder, runner_filter):
+    def check_tf_definition(self, report, root_folder, runner_filter, collect_skip_comments):
```
Comment (schosterbarak) [inline checkov/terraform/runner.py:80]: you probably want to add a default value here or the new param

**Review by nimrodkor — APPROVED** [review]
@schosterbarak Is correct... From the UT:
```
E           TypeError: enrich_definitions_context() missing 1 required positional argument: 'collect_skip_comments'
```

**Pull request merged**